diff --git a/dist/index.mjs b/dist/index.mjs
index c00adcb01d13f05749360eaa4d14730445e394a8..2d3b5c3751fc8ac25605de2fce5f2d9a1d1275e3 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -3,6 +3,6 @@ export { setup } from './chunk-35B2DQBU.mjs';
 import { global } from '@storybook/global';
 import { start } from '@storybook/preview-api';
 
-var{window:globalWindow}=global;globalWindow.STORYBOOK_ENV="vue3";var RENDERER="vue3",api=start(renderToCanvas,{decorateStory,render}),storiesOf=(kind,m)=>api.clientApi.storiesOf(kind,m).addParameters({renderer:RENDERER}),configure=(...args)=>api.configure(RENDERER,...args),{forceReRender}=api,{raw}=api.clientApi;typeof module<"u"&&module?.hot?.decline();
+var{window:globalWindow}=global;globalWindow.STORYBOOK_ENV="vue3";var RENDERER="vue3",api=start(renderToCanvas,{decorateStory,render}),storiesOf=(kind,m)=>api.clientApi.storiesOf(kind,m).addParameters({renderer:RENDERER}),configure=(...args)=>api.configure(RENDERER,...args),{forceReRender}=api,{raw}=api.clientApi;typeof module<"u"&&module?.hot?.decline?.();
 
 export { configure, forceReRender, raw, storiesOf };
diff --git a/src/typings.d.ts b/src/typings.d.ts
deleted file mode 100644
index f4beceae1d634ff3bf08c0fc0005f9dc93e19d6b..0000000000000000000000000000000000000000